src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/operators/Evaluator.scala

```diff
@@ -0,0 +1,23 @@
+package com.yotpo.metorikku.metric.stepActions.dataQuality.operators
+
+case class Evaluator() {
+
+    def sizeAssertion(operator: String, evaluatee: Long): Long => Boolean = operator match {
+        case "==" => {_ == evaluatee}
```

Comment:
Spark also supports === (and it is different from ==)
https://stackoverflow.com/questions/39490236/difference-between-and-in-scala-spark